lib/commands/dev/prepackage.ts

```diff
@@ -22,6 +23,8 @@ export class PrePackageCommand implements ICommand {
 			this.$templatesService.downloadProjectTemplates().wait();
 			this.$logger.info("Downloading item templates.");
 			this.$templatesService.downloadItemTemplates().wait();
+			this.$logger.info("Downloading json schemas");
```

Comment:
This is user-visible message, right? In this case, I recommend we say "Downloading project schemas"